packages/gui/src/components/plot/add/PlotAddSelectFinalDirectory.tsx

```diff
@@ -21,13 +21,14 @@ export default function PlotAddSelectFinalDirectory(props: Props) {
 
   const finalLocation = watch('finalLocation');
   const hasFinalLocation = !!finalLocation;
-  const [defaultFinalDirPath] = useLocalStorage<string>(PlotLocalStorageKeys.FINALDIR);
+  const [defaultFinalDirPath] = usePrefs<string>(PlotLocalStorageKeys.FINALDIR);
```

Comment:
Can we save `setDefaultFinalDirPath` so that we can call `setDefaultFinalDirPath(location);` instead of interacting with the window object?

```suggestion
  const [defaultFinalDirPath, setDefaultFinalDirPath] = usePrefs<string>(PlotLocalStorageKeys.FINALDIR);
```

Reply:
Updated as you suggested.